### ISSUE TITLE
Introducing clear command for taiko

### ISSUE THREAD

**Issue by jackperalta** [getgauge/taiko#26]
Currently when writing commands in the taiko command line interface, the `.code` option includes the whole session with all commands that have been typed in.
This includes possible false positives or failed attempts to perform actions, ergo I would like to see an option (possible `.clear`), which removes all previous commands, so that only the following commands are recorded - or even better, to be able to select the ones, one want to include

This is an example:

> openBrowser()
[PASS] Browser and page initialized
> goto('http://some-ip')
[FAIL] Error: net::ERR_CONNECTION_REFUSED, run `.trace` for more info.
> goto('http://some-ip:8181')
[PASS] Navigated to url "http://some-ip:8181/login"
> write('admin', 'Benutzer')
[FAIL] Error: Element not found, run `.trace` for more info.
> write('admin', $('#username'))
[PASS] Wrote admin into the custom selector $(#username)
> write('xxxx, $('#password'))
[PASS] Wrote xxx into the custom selector $(#password)
// it said PASS, but nothing happened, so I restarted the login
> write('admin', $('#username'))
[PASS] Wrote admin into the custom selector $(#username)
> write('xxx', $('#pwdText'))
[PASS] Wrote xxx into the custom selector $(#pwdText)
> click($('#create'))
[PASS] Clicked custom selector $(#create)

After the last command, when entering `code`, the whole sequence is printed out

**Comment by jackperalta** [comment]
sorry - my first issue and I failed.

Currently when writing commands in the taiko command line interface, the `.code` option includes the whole session with all commands that have been typed in.
This includes possible false positives or failed attempts to perform actions, ergo I would like to see an option (possible `.clear`), which removes all previous commands, so that only the following commands are recorded - or even better, to be able to select the ones, one want to include

This is an example:

> openBrowser()
[PASS] Browser and page initialized
> goto('http://<some-ip>')
[FAIL] Error: net::ERR_CONNECTION_REFUSED, run `.trace` for more info.
> goto('http://<some-ip>:8181')
[PASS] Navigated to url "http://<some-ip>:8181/login"
> write('admin', 'Benutzer')
[FAIL] Error: Element not found, run `.trace` for more info.
> write('admin', $('#username'))
[PASS] Wrote admin into the custom selector $(#username)
> write('xxxx, $('#password'))
[PASS] Wrote xxx into the custom selector $(#password)
// it said PASS, but nothing happened, so I restarted the login
> write('admin', $('#username'))
[PASS] Wrote admin into the custom selector $(#username)
> write('xxx', $('#pwdText'))
[PASS] Wrote xxx into the custom selector $(#pwdText)
> click($('#create'))
[PASS] Clicked custom selector $(#create)

After the last command, when entering `code`, the whole sequence is printed out
